incubator/efs-provisioner/values.yaml

```diff
@@ -0,0 +1,67 @@
+#
+# Default values for EFS provisioner service
+# https://github.com/kubernetes-incubator/external-storage/tree/master/aws/efs
+#
+
+## Deploy environment label, e.g. dev, test, prod
+##
+global:
+  deployEnv: dev
+
+## Containers
+##
+replicaCount: 1
+revisionHistoryLimit: 10
+image:
+  repository: quay.io/external_storage/efs-provisioner
+  tag: v0.1.2
+  pullPolicy: IfNotPresent
+
+busyboxImage:
+  repository: gcr.io/google_containers/busybox
+  tag: 1.27
+  pullPolicy: IfNotPresent
+
+## Configure provisioner
+## https://github.com/kubernetes-incubator/external-storage/tree/master/aws/efs#deployment
+##
+efsProvisioner:
+  efsFileSyteId: fs-12345678
```

Comment:
`efsFileSyteId` -> `efsFileSystemId`

Reply:
Thanks @nazarewk! I have made the change.